lib/game_teams.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'game_team'
+require 'csv'
```

Comment:
this is the class gameteams that creates ALL game teams from csv.

Reply:
I'm wondering if this is redundant or necessary for stat tracker to do its job

Reply:
We'll find out...??